src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RC2.cs

```diff
@@ -3,10 +3,12 @@
 
 using Internal.Cryptography;
```

Comment:
very very much a nit:  This does not seem to follow all the previous style changes that were made.

Really think these style changes should have been done in a separate PR.

Reply:
Done